integration/combination/test_api_with_authorizers.py

```diff
@@ -1,10 +1,14 @@
+from unittest.case import skipIf
+
 import requests
 
 from integration.helpers.base_test import BaseTest
 from integration.helpers.deployer.utils.retry import retry
 from integration.helpers.exception import StatusCodeError
+from integration.helpers.resource import current_region_does_not_support
 
 
+@skipIf(current_region_does_not_support(["Cognito"]), "Cognito is not supported in this testing region")
```

Comment:
I wonder whether we should put service strings like "Cognito" into constants so that there will be less likely to have typos in the future.

Reply:
Updated with using constant as service name